test/quicktest.jl

```diff
@@ -0,0 +1,43 @@
+using Revise
+using DataDrivenDiffEq
+using LinearAlgebra
+using ModelingToolkit
+using OrdinaryDiffEq
+using Plots 
+using Statistics
+using StatsBase
+using Random
+
+@variables u[1:2]
+basis = Basis([polynomial_basis(u, 5); sin.(u); cos.(u)], u)
+
+function pendulum(u, p, t)
+    x = u[2]
+    y = -9.81sin(u[1]) - 0.1u[2]^3 - 0.2 * cos(u[1])
+    return [x;y]
+end
+
+u0 = [0.99π; -1.0]
+tspan = (0.0, 20.0)
+dt = 0.1
+prob = ODEProblem(pendulum, u0, tspan)
+sol = solve(prob, Tsit5(), saveat=dt)
+
+
+
+
+X = Array(sol)
+t = sol.t
+Random.seed!(1234)
+X = X .+ 1e-1*randn(size(X))
+
+dd_prob_noisy = ContinuousDataDrivenProblem(
+    X, t, GaussianKernel()
```

Comment:
where is the kernel used?

Reply:
In `src/problem/types.jl` L 206 / 227 . 

```Julia
function ContinuousDataDrivenProblem(X::AbstractMatrix, t::AbstractVector, collocation = InterpolationMethod(); kwargs...)
```

Just a copy paste from the DiffEqFlux collocation stuff + DataInterpolations

Reply:
This is my local testfile btw, so feel free to ignore.

Reply:
> Just a copy paste from the DiffEqFlux collocation stuff + DataInterpolations

Ahh I see, makes sense.